modules/javafx.graphics/src/main/native-glass/mac/GlassView.m

```diff
@@ -35,8 +35,11 @@
 #import "GlassMacros.h"
 #import "GlassWindow.h"
 #import "GlassView3D.h"
+#import "GlassViewMTL3D.h"
```

Comment:
GlassViewMTL3D.h, GlassViewCGL3D.h,  GlassLayerMTL3D.h, GlassLayerCGL3D.h imports can be removed.

Reply:
This still needs to be resolved.

Reply:
Updated